config-connector/solutions/iam/kpt/service-account/README.md

```diff
@@ -0,0 +1,102 @@
+Service Account
+==================================================
+
+# NAME
+
+  service-account
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a service account in your desired project, and grant it a specific role (default to iam.serviceAccountKeyAdmin) on the service account resource level.
+
+  You can also create any number of service accounts and grant them any number of roles by adding new or updating the config YAMLs and then applying them.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/member-iam member-iam
+  ```
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a service account
+  (`cnrm-system@[PROJECT_ID].iam.gserviceaccount.com`) with the following
```

Comment:
"using a service account (`cnrm-system@[PROJECT_ID].iam.gserviceaccount.com`)" -> "using the cnrm-system service account"

"Config Connector cnrm-system service account" is the latest agreed way to refer to config connector's run as sa.

Reply:
Fixed. Thanks!